### PR TITLE
[iOS10, UIKit] More work

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/uikit.cs [hunk 1]
```diff
@@ -15492,6 +15569,53 @@ public interface UIPressesEvent {
...
+		[Abstract]
+		[Export ("previewInteractionDidCancel:")]
+		void DidCancel (UIPreviewInteraction previewInteraction);
+
+		[Export ("previewInteractionShouldBegin:")]
+		[DelegateName ("Func<UIPreviewInteraction,bool>"), DefaultValue(true)]
+		bool ShouldBegin (UIPreviewInteraction previewInteraction);
+
+		[Export ("previewInteraction:didUpdateCommitTransition:ended:")]
+		[EventArgs ("NSPreviewInteractionPreviewUpdate")]
```
Comment (dalexsoto) [inline src/uikit.cs]: For some reason this won't show up here `[EventArgs ("NSPreviewInteractionPre` but in the file it is complete

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**